common/errors.js

```diff
@@ -194,17 +199,21 @@
                 window.location.reload();
             };
 
-            modalInstance.result.then(function () {
+            modalInstance.result.then(function (actionBtnIdentifier) {
+
                 if (errorCode == errorNames.unauthorized && !providedLink) {
                     var x = window.innerWidth/2 - 800/2;
                     var y = window.innerHeight/2 - 600/2;
 
                     var win = window.open("", '_blank','width=800,height=600,left=' + x + ',top=' + y);
                     Session.loginInAPopUp(win, reloadCb);
-                }else if(errorCode == errorNames.conflict){
-                   $window.open(redirectLink, '_blank');
                 } else {
-                    $window.location.replace(redirectLink);
+                    if(actionBtnIdentifier == "reload"){
+                        reloadCb();
+                    } else{
```

Comment:
Add a comment for this else case saying that it handles the case when the modal is closed without a result. This is the case when the `ok` button is selected (the default case).